source/DataTracker.mc

```diff
@@ -3,112 +3,129 @@ using Toybox.ActivityMonitor as Act;
 //! Class that holds the data that has to be
 //! valculated and is displayed in the UI.
 class DataTracker {
-	//! Points to win a set
-	hidden var SCORE_FOR_SET = 11;
-	//! Number of steps done during the activity
-	hidden var numberOfSteps;
-	//! Number of calories burned during the activity
-	hidden var numberOfCalories;
-	//! Number of steps done when the activity started
-	hidden var initialSteps;
-	//! Amount of calories burnt until the activity started
-	hidden var initialCalories;
-	//! Session used to record the activity
-	hidden var session;
-	//! Sets tracker
-	hidden var setsTracker;
-	
-	//! Constructor
-	function initialize() {
-		session = new Session();
-		restart();
-	}
-	
-	//! Restarts all the data field to make it ready
-	//! to record a new session
-	function restart() {
-		numberOfSteps = 0;
-		numberOfCalories = 0;
+    //! Points to win a set
+    hidden var SCORE_FOR_SET = 89;

```

Comment:
To follow coding conventions, constants should be in UPPER-CASE and variables should be in lower-case. Since you converted a constant to a variable, it should be renamed to use lower-case.